src/community/resources-and-tools/index.md.njk

```diff
@@ -73,6 +73,9 @@ GOV.UK Frontend compatible React components.
 [GOV.UK Design System Formbuilder](https://github.com/DFE-Digital/govuk_design_system_formbuilder) -
 A Rails form builder using GOV.UK Frontend.
 
+[GOV.UK Components](https://github.com/DFE-Digital/govuk-components) -
+A lightweight set of Rails components using GOV.UK Frontend.
```

Comment:
Thanks @hannalaakso, @peteryates! I can't think of a better way to say 'non-form' without it getting clunky so I'm all good to keep it as is. 

I'd perhaps suggest adding a sentence that it's meant to be used with Formbuilder - up to you both though.

> GOV.UK components - A set of non-form Rails components using GOV.UK Frontend. You can use it together with the Design System Formbuilder.

Reply:
Maybe `DfE Design System Formbuilder`?

Reply:
Well, the issue there is that the link to the Formbuilder doesn't mention DfE, so if possible we'd want to match with whatever we say on the page.

Fair point on adding contributor info to the listings though, something for me to look at for improving this page.